supporting_scripts/course-scripts/quick-course-setup/README.md

```diff
@@ -0,0 +1,132 @@
+# Artemis Course Setup Scripts
+
+This project contains Python scripts that automate the setup and management of courses and users in an Artemis instance. The scripts help you create a course, generate users, enroll them into courses, and manage programming exercises, including participation and commits. The project is useful for setting up test environments or quickly initializing a course with a large number of students.
+
+## Setup
+
+### 1. Install Python and the Python Plugin for IntelliJ
+
+- Ensure that Python (preferably version 3.7 or 3.9) is installed on your system.
+- Install the [Python Plugin for IntelliJ](https://plugins.jetbrains.com/plugin/631-python).
+- Enable Python support in IntelliJ:
+  - Go to `File > Project Structure > Facets > Add - Python`.
+  - Add a Python environment by configuring the Python interpreter.
+  - Add a module in IntelliJ by navigating to `File > Project Structure > Modules > Add - Python`.
+
+### 2. Configure the Environment
+
+- Start your local Artemis instance.
+- Configure the `config.ini` file according to your local or test server setup. This includes settings like `server_url`, `client_url`, `admin_user`, `admin_password`, and course-specific configurations.
```

Comment:
I think we should directly link the config file and push a version with default settings. Why should every developer have to create the config file, think about where to create it and which values need to be present? 
This can be prevented by simply supplying a default configuration
```suggestion
- Configure the [config.ini](./config.ini) file according to your local or test server setup.
```

Reply:
Why do we not directly link the file? 

Why do we need to list variables that can be configured if they can be seen in the linked `config.ini` anyways?